### PR TITLE
Royhayward

### PR DESCRIPTION
**Pull request by royhayward**
Just pulling your leg.... :rocket:

### CODE REVIEW

File: githubuser-roy-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Roy Hayward's Bio
+
+- Roy Hayward roy@theintegrationengineer.com
```
Comment (obscurecloud) [inline githubuser-roy-bio.md:3]: I think you spelled your name wrong

**Comment by kentkingdon** [comment]
What's your middle name though?

File: githubuser-roy-bio.md (hunk 1 above)
Comment (kentkingdon) [inline githubuser-roy-bio.md:3]: What's your middle name though?

Comment (royhayward) [inline githubuser-roy-bio.md:3]: @kentkingdon Middle Name Added.

**Pull request merged**